nifi-nar-bundles/nifi-extension-utils/nifi-record-utils/nifi-avro-record-utils/src/main/java/org/apache/nifi/schema/access/SchemaAccessUtils.java

```diff
@@ -45,22 +44,24 @@
             "The FlowFile contains 3 Attributes that will be used to lookup a Schema from the configured Schema Registry: 'schema.identifier', 'schema.version', and 'schema.protocol.version'");
     public static final AllowableValue INHERIT_RECORD_SCHEMA = new AllowableValue("inherit-record-schema", "Inherit Record Schema",
         "The schema used to write records will be the same schema that was given to the Record when the Record was created.");
+    public static final AllowableValue CONFLUENT_ENCODED_SCHEMA = new AllowableValue("confluent-encoded", "Confluent Content-Encoded Schema Reference",
```

Comment:
I think we need to be clear which version of the confluent schema registry we support.  If a later version changes the magic bytes or anything like that and assumes the clients will handle it we might need to adjust our handling.